### PR TITLE
feat(admin): add perunobserver

### PR DESCRIPTION
**Pull request by metodej**
* added role perunobserver, this role should be able to read everything perunadmin can

### CODE REVIEW

**Review by HejdaJakub — APPROVED** [review]

File: libs/perun/services/src/lib/gui-auth-resolver.service.ts [hunk 1]
```diff
@@ -199,19 +204,23 @@ export class GuiAuthResolver {
   }
 
   public canManageFacilities(): boolean {
-    return this.hasAtLeasOne(Role.PERUNADMIN, Role.FACILITYADMIN, Role.FACILITIYOBSERVER);
+    return this.hasAtLeasOne(Role.PERUNADMIN, Role.PERUNOBSERVER, Role.FACILITYADMIN, Role.FACILITIYOBSERVER);
```
Comment (xkureck) [inline libs/perun/services/src/lib/gui-auth-resolver.service.ts:207]: with this change there could be probably some change in the name of the method

**Review by xkureck — CHANGES_REQUESTED** [review]
changes in this PR should be OK, but imcomplete
in the PERUN ADMIN section there is no restriction for perun observer for changing/deleting action - this can be done by some gui member in other PR if you dont feel like doing it

File: apps/admin-gui/src/app/admin/pages/admin-page/admin-services/admin-services.component.html [hunk 2]
```diff
@@ -2,6 +2,7 @@
   <h1 class="page-subtitle">{{'ADMIN.SERVICES.TITLE' | translate}}</h1>
   <perun-web-apps-refresh-button (refresh)="refreshTable()"></perun-web-apps-refresh-button>
   <button
+    *ngIf="authResolver.isPerunAdmin()"
```
Comment (xkureck) [inline apps/admin-gui/src/app/admin/pages/admin-page/admin-services/admin-services.component.html:5]: on this page also put this if on the delete button please 
image

**Review by xkureck — CHANGES_REQUESTED** [review]
on component user-identities please add also check in rights to add and remove button
on component user-settings-service-identities please also add check in right to add/remove button

File: apps/admin-gui/src/app/admin/pages/admin-page/admin-services/admin-services.component.html (hunk 2 above)
Comment (metodej) [reply apps/admin-gui/src/app/admin/pages/admin-page/admin-services/admin-services.component.html:5]: Fixed, thanks.

**Review by xkureck — APPROVED** [review]

**Pull request merged**